src/core/internal/string.d

```diff
@@ -27,6 +27,26 @@ char[] unsignedToTempString(ulong value, char[] buf, uint radix) @safe
     return buf[i .. $];
 }
 
+auto unsignedToTempString(ulong value, uint radix) @safe
+{
+    struct Result
+    {
+        // need to handle 65 bytes because radix could be 2, and we
+        // could be signed.
```

Comment:
It's used by `signedToTempString` instead of duplicating the code and creating a new type.